src/main/resources/assets/immersiveweapons/lang/zh_cn

```diff
@@ -0,0 +1,687 @@
+	"item.immersiveweapons.obsidian_rod": "黑曜石杆",
+	"item.immersiveweapons.molten_pickaxe": "熔融镐",
+	"item.immersiveweapons.molten_axe": "熔融斧",
+	"item.immersiveweapons.molten_shovel": "熔融铲",
+	"item.immersiveweapons.molten_hoe": "熔融锄",
+	"item.immersiveweapons.electric_ingot": "电石锭",
+	"item.immersiveweapons.conductive_alloy": "导电合金",
+	"item.immersiveweapons.tesla_ingot": "特斯拉锭",
+	"item.immersiveweapons.copper_sword": "铜剑",
+	"item.immersiveweapons.copper_pickaxe": "铜镐",
+	"item.immersiveweapons.copper_axe": "铜斧",
+	"item.immersiveweapons.copper_shovel": "铜铲",
+	"item.immersiveweapons.copper_hoe": "铜锄",
+	"item.immersiveweapons.molten_helmet": "熔融头盔",
+	"item.immersiveweapons.molten_chestplate": "熔融胸甲",
+	"item.immersiveweapons.molten_leggings": "熔融护腿",
+	"item.immersiveweapons.molten_boots": "熔融靴子",
+	"item.immersiveweapons.copper_helmet": "铜头盔",
+	"item.immersiveweapons.copper_chestplate": "铜胸甲",
+	"item.immersiveweapons.copper_leggings": "铜护腿",
+	"item.immersiveweapons.copper_boots": "铜靴子",
+	"item.immersiveweapons.wooden_tool_rod": "木制工具杆",
+	"item.immersiveweapons.copper_pike": "铜枪",
+	"item.immersiveweapons.copper_pike_head": "铜枪头",
+	"item.immersiveweapons.copper_nugget": "铜粒",
+	"item.immersiveweapons.copper_arrow": "铜箭",
+	"item.immersiveweapons.tesla_helmet": "特斯拉头盔",
+	"item.immersiveweapons.tesla_chestplate": "特斯拉胸甲",
+	"item.immersiveweapons.tesla_leggings": "特斯拉护腿",
+	"item.immersiveweapons.tesla_boots": "特斯拉靴子",
+	"item.immersiveweapons.iron_pike": "铁枪",
+	"item.immersiveweapons.iron_pike_head": "铁枪头",
+	"item.immersiveweapons.diamond_pike": "钻石枪",
+	"item.immersiveweapons.diamond_pike_head": "钻石枪头",
+	"item.immersiveweapons.diamond_shard": "钻石碎片",
+	"item.immersiveweapons.netherite_pike": "下界合金枪",
+	"item.immersiveweapons.stone_pike": "石枪",
+	"item.immersiveweapons.stone_shard": "石头碎片",
+	"item.immersiveweapons.stone_pike_head": "石枪头",
+	"item.immersiveweapons.wooden_pike": "木枪",
+	"item.immersiveweapons.wooden_pike_head": "木枪头",
+	"item.immersiveweapons.wooden_shard": "木头碎片",
+	"item.immersiveweapons.golden_pike": "金枪",
+	"item.immersiveweapons.golden_pike_head": "金枪头",
+	"item.immersiveweapons.iron_arrow": "铁枪",
+	"item.immersiveweapons.diamond_arrow": "钻石箭",
+	"item.immersiveweapons.golden_arrow": "金箭",
+	"item.immersiveweapons.stone_arrow": "石箭",
+	"item.immersiveweapons.wooden_arrow": "木箭",
+	"item.immersiveweapons.netherite_arrow": "下界合金箭",
+	"item.immersiveweapons.flintlock_pistol": "燧发枪",
+	"item.immersiveweapons.copper_musket_ball": "铜制弹丸",
+	"item.immersiveweapons.wooden_musket_ball": "木制弹丸",
+	"item.immersiveweapons.stone_musket_ball": "石制弹丸",
+	"item.immersiveweapons.iron_musket_ball": "铁制弹丸",
+	"item.immersiveweapons.golden_musket_ball": "金制弹丸",
+	"item.immersiveweapons.diamond_musket_ball": "钻石制弹丸",
+	"item.immersiveweapons.netherite_musket_ball": "下界合金制弹丸",
+	"item.immersiveweapons.flintlock_assembly": "燧发机组件",
+	"item.immersiveweapons.trigger_assembly": "扳机组件",
+	"item.immersiveweapons.heavy_wooden_stock": "重型木托",
```

Comment:
```
Heavy Wooden Tray
```
This is a stock not a tray. It is the object on a gun that rests against your shoulder.